.github/workflows/deploy.yaml

```diff
@@ -38,6 +38,11 @@ on:
         required: false
         default: "false"
         type: string
+      lockIfManualDeploy:
```

Comment:
Nit: As per this [comment](https://github.com/alphagov/frontend/pull/3289#discussion_r919913419) - maybe this should be called "Disable continuous deployment"